### PR TITLE
Update egg-squad.json

### PR DESCRIPTION
**Pull request by Cartitarul**
Fixed some bugs

# Description

Fixed some mods that kept downloading bug and the required parameter on the workshop ids

## Checklist for all submissions

* [x ] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x ] Have you tested and reviewed your changes with confidence that everything works?
* [x ] Did you branch your changes and PR from that branch and not from your master branch?
* [x ] You verify that the start command applied does not use a shell script
  * [x ] If some script is needed then it is part of a current yolk or a PR to add one
* [x ] The egg was exported from the panel

### CODE REVIEW

File: squad/egg-squad.json [hunk 1]
```diff
@@ -157,8 +157,8 @@
             "default_value": "",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|string",
```
Comment (QuintenQVD0) [inline squad/egg-squad.json:160]: then make this `nullable|string`

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]
Just this one little thing, then it is fine

**Comment by Cartitarul** [comment]
Done. I have the egg for windows. I will try to do a request later this day for it as well

**Review by QuintenQVD0 — APPROVED** [review]
LGTM! thank you.

**Pull request merged**